package.json

```diff
@@ -3,7 +3,7 @@
   "title": "React Native Stripe Payments",
   "version": "1.0.6",
   "description": "Lightweight, easy to integrate and use React native library for Stripe payments (using Payment Intents) compliant with SCA (strong customer authentication).",
-  "main": "build/index.js",
+  "main": "lib/index.tsx",
```

Comment:
@j-bin is this change is needed?

Reply:
@wynnej1983  Causing 'build/index.js' missing error. If you yarn and pack it, you won't get error.